### PR TITLE
[content structure] Adjusted inline display

### PR DESCRIPTION
**Pull request by Tjocksmurf**

### CODE REVIEW

**Comment by Tjocksmurf** [comment]
[Confirm structure of content displayed by Reference paragraphs](https://trello.com/c/7XUuvxvB/355-confirm-structure-of-content-displayed-by-reference-paragraphs)

File: config/default/core.entity_view_display.node.h_contact.h_inline.yml [hunk 1]
```diff
@@ -50,14 +50,6 @@ targetEntityType: node
...
 content:
-  field_h_contact_title_role:
-    type: string
-    weight: 2
-    region: content
-    label: hidden
-    settings:
-      link_to_entity: false
-    third_party_settings: {  }
   field_h_contact_type:
```
Comment (Benjen) [inline config/default/core.entity_view_display.node.h_contact.h_inline.yml]: Contact type should be removed from this display mode.

File: config/default/core.entity_view_display.node.h_publication.h_inline.yml [hunk 2]
```diff
@@ -55,9 +55,17 @@ content:
       image_style: h_thumbnail_max_width
       image_link: ''
     third_party_settings: {  }
+  field_h_publication_type:
+    type: entity_reference_label
+    weight: 2
+    region: content
+    label: above
```
Comment (Benjen) [inline config/default/core.entity_view_display.node.h_publication.h_inline.yml]: Hide label

File: config/default/core.entity_view_display.node.h_publication.h_inline.yml [hunk 3]
```diff
@@ -55,9 +55,17 @@ content:
       image_style: h_thumbnail_max_width
       image_link: ''
     third_party_settings: {  }
+  field_h_publication_type:
+    type: entity_reference_label
+    weight: 2
```
Comment (Benjen) [inline config/default/core.entity_view_display.node.h_publication.h_inline.yml]: Place between heading and summary.

**Review by Benjen — CHANGES_REQUESTED** [review]
Hi @Tjocksmurf ,

See comments for information on suggested changes.

**Review by Benjen — APPROVED** [review]

**Pull request merged**